src/Server.php

```diff
@@ -53,11 +53,11 @@ class Server
 
     /**
      * Create Server instance.
-     * @param FilesystemInterface $source The source file system.
-     * @param FilesystemInterface $cache  The cache file system.
-     * @param ApiInterface        $api    The image manipulation API.
+     * @param FilesystemInterface      $source The source file system.
+     * @param FilesystemInterface|null $cache  The cache file system.
+     * @param ApiInterface             $api    The image manipulation API.
      */
-    public function __construct(FilesystemInterface $source, FilesystemInterface $cache, ApiInterface $api)
+    public function __construct(FilesystemInterface $source, $cache, ApiInterface $api)
```

Comment:
+1 for the above suggestion of having the type hint on the $cache argument with a default value of null. I.e.

```php
public function __construct(FilesystemInterface $source, FilesystemInterface $cache = null, ApiInterface $api)
```